### ISSUE TITLE
Feature: source pr infos not from merge message

### ISSUE THREAD

**Issue by eksrha** [fullstack-devops/awesome-ci#58]
```javascript
 const prs = await repo.listPullRequests(
        { "state": "all", "sort": "updated", "direction": "desc", "per_page": "10" })
    const pull = prs.data.find(p => p.merge_commit_sha === sha1);
```

**Comment by eksrha** [comment]
resolved with #69

**Comment by eksrha** [comment]
@kiwo feature is in version 0.9.1
